features/features.go

```diff
@@ -345,6 +345,14 @@ var (
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
+	FeatureGateAzureClusterHostedDNS = newFeatureGate("AzureClusterHostedDNS").
```

Comment:
```suggestion
	FeatureGateAzureClusterHostedDNSInstall = newFeatureGate("AzureClusterHostedDNSInstall").
```

Needed for test analyzer

Reply:
Done.